Update daos-overview.md several updates (#1060)
argonne-lcf/user-guides: docs/aurora/data-management/daos/daos-overview.md

is almost always the no-vni issue or network issue and not a DAOS issue

### 5. `na_ofi_mem_register` errors

> There is a network limitation where if your application does alot of IO from a memory buffer that is severly segmented you may see an error like this:

Reviewer: ```suggestion
There is a network limitation where if your application does a lot of IO from a memory buffer that is severely segmented you may see an error like this:
```

```bash linenums="1"
11/25-21:17:21.87 x4305c3s5b0n0 DAOS[75672/75672/0] external ERR  # [7284.159605] mercury->mem [error] /home/daos/pre/build/external/release/mercury/src/na/na_ofi.c:8870 na_ofi_mem_register() fi_mr_enable() failed, rc: -28 (No space left on device), mr_reg_count: 15677
```

> This error may be gotten around by the following environment variable setting at run time:

Reviewer: ```suggestion
A workaround for this error by setting the following environment variable at runtime:
```
